Frontend/Gmfctn/src/app/modules/+auth/services/authenticate/authenticate.service.ts

```diff
@@ -37,4 +38,33 @@ export class AuthenticateService implements OnDestroy{
         this.tokens$.next(newTokens);
       }, catchError( (err: HttpErrorResponse) => of(err))), take(1));
   }
+
+  isLoggedIn(): boolean {
+    if (this.isTokenStillActive())
+    {
+      const tokens: Tokens = {
+        token: localStorage.getItem('AToken') ?? '',
+        aTTL: localStorage.getItem('UTC_TTL') ?? '',
+        refreshToken: localStorage.getItem('RToken') ?? '',
+      };
+
+      if (tokens.token === '' || tokens.aTTL === '' || tokens.refreshToken === '') {
+        return false;
+      }
+      else {
+        this.tokens$.next(tokens);
+
+        return true;
+      }
+    }
+
+    return false;
+  }
+
+  isTokenStillActive(): boolean {
+    const expiringDate = new Date(new Date(localStorage.getItem('UTC_TTL') + '').toUTCString());
+    const now = new Date(new Date().toUTCString());
+
+    return expiringDate > now;
```

Comment:
Why do you need this conversion to UTC string?
It'd be much easier to use timestamps 
```
    const expiringDate = new Date(localStorage.getItem('UTC_TTL')).getTime();
    const now = Date.now();
```